build.gradle.kts

```diff
@@ -1,9 +1,5 @@
 buildscript {
     val jacocoVersion by rootProject.extra("0.8.9")
-    val minAndroidVersion by rootProject.extra { 29 }
```

Comment:
Making it consistent across the repositories; this script now matches:
https://github.com/govuk-one-login/mobile-android-secure-store/blob/main/build.gradle.kts

These values are now set by the APKConfig object:
https://github.com/govuk-one-login/mobile-android-authentication/blob/ci/migrate-to-reusable-action/buildLogic/plugins/src/main/kotlin/uk/gov/authentication/config/ApkConfig.kt
https://github.com/govuk-one-login/mobile-android-secure-store/blob/8cc550c38885186bc32b09b69e31ca85a92104d6/buildLogic/plugins/src/main/kotlin/uk/gov/securestore/config/ApkConfig.kt

If we prefer to keep them in the build.gradle.kts, we can.. but probably should do it everywhere. I'm trying to minimise the complexity of our code so that it's easier to learn once and understand everywhere.